### PR TITLE
Adds fixture for orphans_api_client

### PR DESCRIPTION
**Pull request by dkliban**
[noissue]

### CODE REVIEW

File: pulpcore/tests/functional/__init__.py [hunk 1]
```diff
@@ -212,6 +213,11 @@ def redirect_contentguard_api_client(pulpcore_client):
     return ContentguardsContentRedirectApi(pulpcore_client)
 
 
+@pytest.fixture
+def orphans_api_client(pulpcore_client):
+    return OrphansApi(pulpcore_client)
```
Comment (mdellweg) [inline pulpcore/tests/functional/__init__.py:218]: That's the deprecated interface, no?
Can we leave a comment / docstring to not forget?

Comment (dkliban) [reply pulpcore/tests/functional/__init__.py:218]: yes it is. i made a comment about it in the test, but i'll add one here also.

**Review by mdellweg — APPROVED** [review]

File: pulpcore/tests/functional/__init__.py (hunk 1 above)
Comment (bmbouter) [reply pulpcore/tests/functional/__init__.py:219]: Can we move this fixture to just the module where it's used? I don't think we want to make the deprecated one widely available. If you put the fixture up at the top where it's used it'll work.

Comment (dkliban) [reply pulpcore/tests/functional/__init__.py:219]: That's fine with me.

**Comment by dkliban** [comment]
I am closing this PR in favor of adding it to the pulp_file test that is using it.

**Pull request closed without merging**